dns-policy-operator/charmcraft.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+name: dns-policy
+type: charm
+subordinate: true
+title: DNS policy charm
+summary: A very short one-line summary of the charm.
+
+description: |
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+config:
+  options:
+    # An example config option to customise the log level of the workload
+    log-level:
+      description: |
+        Configures the log level of gunicorn.
+        Acceptable values are: "info", "debug", "warning", "error" and "critical"
+      default: "info"
+      type: string
+    debug:
```

Comment:
Can we merge this with the previous option? So enable debug mode if the above option is set to "debug" or "info"?

Reply:
Previous section is removed